.github/workflows/docker-action.yml

```
@@ -18,17 +18,22 @@ jobs:
       matrix:
         image:
           - 'mathcomp/mathcomp-dev:coq-dev'
+          - 'mathcomp/mathcomp:2.0.0-coq-8.17'
+          - 'mathcomp/mathcomp:1.17.0-coq-8.17'
+          - 'mathcomp/mathcomp:1.16.0-coq-8.17'
+          - 'mathcomp/mathcomp:1.15.0-coq-8.16'
           - 'mathcomp/mathcomp:1.14.0-coq-8.15'
           - 'mathcomp/mathcomp:1.13.0-coq-8.15'
           - 'mathcomp/mathcomp:1.12.0-coq-8.14'
       fail-fast: false
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: coq-community/docker-coq-action@v1
         with:
-          opam_file: 'coq-autosubst-ci.opam'
+          opam_file: 'coq-autosubst.opam'
```

```suggestion
          opam_file: 'coq-autosubst-ci.opam'
```
If you use the standard opam file, `coq-autosubst.opam`, then the test suite will not be run in CI. For example, this means that the SSR examples are not checked.